packages/components/Button/src/Button.types.ts

```diff
@@ -42,6 +43,11 @@ export interface ButtonCoreTokens extends LayoutTokens, FontTokens, IBorderToken
    */
   spacingIconContentAfter?: number;
 
+  /**
+   * The depth of the shadow of the button
+   */
+  shadowDepth?: ShadowDepth;
```

Comment:
I wonder if we have some public facing documentation describing our shadow ramp, that we could link here.

Reply:
Nothing that I'm aware of but was planning to write Shadow documentation and think that would make a lot of sense to add there

Reply:
It could be part of the SPEC.md file as a start. More detailed documentation is probably a designer problem

Reply:
Made a task for myself to write documentation